linalg/sundials.hpp

```diff
@@ -547,6 +686,10 @@ class ARKStepSolver : public ODESolver, public SundialsSolver
 
    /// Destroy the associated ARKode memory and SUNDIALS objects.
    virtual ~ARKStepSolver();
+
+   virtual MemoryClass GetMemoryClass() const
+   { return Device::GetMemoryClass(); }
```

Comment:
Should this be in the ` SundialsSolver` class?

Reply:
This code is unused, so I removed it entirely.